### ISSUE TITLE
normalmap appears incorrect on some meshes

### ISSUE THREAD

**Issue by djmaesen** [godotengine/godot#65713]
### Godot version

4.0 alpha16

### System information

linux

### Issue description

some of my models have strange normal map behaviour, 
i think this might be with the new normal/tangent compression since alpha 15

### Steps to reproduce

have a look at my screenshot
![normalerror](https://user-images.githubusercontent.com/95492794/189712277-0059446a-1744-4973-bfcb-b7b5ad4b925b.jpg)

**Comment by Calinou** [comment]
Can you reproduce this in 4.0.alpha14 or older alphas?

Also, please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by clayjohn** [comment]
I ran the test project and reproduced the errors. The errors went away when I reimported the mesh. 

There is a connected usability issue here though (that is reported and tracked in https://github.com/godotengine/godot/issues/64854) Users who import files, then extract the ArrayMesh and save it into a TSCN format don't benefit from reimporting their meshes which makes changes to the ArrayMesh format extra dangerous for them. 

Closing this now as all outstanding issues should be tracked in https://github.com/godotengine/godot/issues/64854
